readersOweReading: false temper/temper-lib-scrollable-menu/scrollable-menu-buttongroup/scrollable-menu-buttongroup.module.ts
diff --git a/bun.lock b/bun.lock
index f7e49eba4d60238b190b38709cbb11d658970673..a11197ed2f0aba20f63cb4eb382ca87eb1e7ff59 100644
--- a/bun.lock
+++ b/bun.lock
@@ -2433,6 +2433,9 @@
     },
     "temper/temper-lib-scrollable-menu": {
       "name": "@akasha/temper-lib-scrollable-menu",
+      "dependencies": {
+        "@akasha/code-system": "workspace:@akasha/code@*",
+      },
     },
     "temper/temper-lib-sets": {
       "name": "@akasha/temper-lib-sets",
diff --git a/temper/temper-lib-scrollable-menu/package.json b/temper/temper-lib-scrollable-menu/package.json
index 899d46c94733bf97c8451a735eefeac058c63b50..296d24eb6d0ac3dd3809ecde9f27929569a508c1 100644
--- a/temper/temper-lib-scrollable-menu/package.json
+++ b/temper/temper-lib-scrollable-menu/package.json
@@ -75,5 +75,8 @@
     "./util-menu": "./scrollable-menu-util-menu/scrollable-menu-util-menu.module.code.ts",
     "./util-tooltip": "./scrollable-menu-util-tooltip/scrollable-menu-util-tooltip.module.code.ts",
     "./util-visible": "./scrollable-menu-util-visible/scrollable-menu-util-visible.module.code.ts"
+  },
+  "dependencies": {
+    "@akasha/code-system": "workspace:@akasha/code@*"
   }
 }
diff --git a/temper/temper-lib-scrollable-menu/scrollable-menu-buttongroup/scrollable-menu-buttongroup.module.ts b/temper/temper-lib-scrollable-menu/scrollable-menu-buttongroup/scrollable-menu-buttongroup.module.ts
index 52a7ae06fb6387cb2e4ddc0f2705975e9e78a070..05ec4b73b1788695d5c9e094066a7bd7558747ca 100644
--- a/temper/temper-lib-scrollable-menu/scrollable-menu-buttongroup/scrollable-menu-buttongroup.module.ts
+++ b/temper/temper-lib-scrollable-menu/scrollable-menu-buttongroup/scrollable-menu-buttongroup.module.ts
@@ -17,7 +17,7 @@ export const scrollableMenuButtongroup = {
     },
     {
       invariantKind: "departure",
-      statement: "Check-all and invert are carried out by driving each button's original handler.",
+      statement: "A bulk check or invert is carried out by driving each button's original handler.",
     },
     {
       invariantKind: "departure",
